src/extension.ts

```diff
@@ -43,6 +59,37 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
 			}
 			return stopClients();
 		}),
+		vscode.commands.registerCommand('terraform.init', async () => {
+			const selected = await vscode.window.showOpenDialog({
+				canSelectFiles: false,
+				canSelectFolders: true,
+				canSelectMany: false,
+				defaultUri: vscode.workspace.workspaceFolders[0].uri,
+				openLabel: "Initialize"
+
+			});
+			if (selected) {
```

Comment:
nitpick, can `selected` be nil, but `showOpenDialog` didn't throw?

Reply:
Yes, if the user hits cancel on the file finder, it's not an error there's just no value.